machine/wnc/wnc_osw1800/machine.make

```diff
@@ -0,0 +1,75 @@
+# WNC ONIE
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bft
+
+VENDOR_REV ?= 0
+
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# The VENDOR_VERSION string is appended to the overal ONIE version
+# string.  HW vendors can use this to appended their own versioning
+# information to the base ONIE version string.
+VENDOR_VERSION = .OSW1800.1.01
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Open Compute Project IANA number
+# WNC is 15756
+VENDOR_ID = 15756
+
+# Skip the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+
+# The onie-syseeprom command in i2ctools is deprecated.  It is recommended to
+# use the one implemented in busybox instead.  The option intends to provide a
+# quick way to turn off the feature in i2ctools.  The command will be removed
+# from i2ctools in the future once all machines migrate their support of
+# sys_eeprom to busybox.
+#
+# The option is significant when I2CTOOLS_ENABLE is 'yes'
+#
+I2CTOOLS_SYSEEPROM = no
```

Comment:
you can remove this setting (and the comment above it) as this is default since July 2017.

Reply:
I updated it. Thanks.